README.md

```diff
@@ -45,17 +45,17 @@ Let's call this `example.json` and fill it out:
 
 ```
 
-Once a configuraion file is complete, the file can be submitted as parameter to the program where the engine executes the code. The `run_analysis` module executes the code:
+Once a configuration file is complete, the file can be submitted to the `analysis_schema` command line interface,
 
 ```
-python3 run_analysis.py example.py
+analysis_schema run-analysis example.json
 ```
 
-After running that line of code in the command line, image files of a projection plot or other visauls are returned.
+After running that command, image files will be saved and their filenames will display at the command line.
```

Comment:
this section is the only thing i actually changed intentionally in the readme... the rest of the changes were my editor auto-removing some end of line spaces.